docs/snippets/modules/data_connection/vectorstores/get_started.mdx

```diff
@@ -116,3 +116,4 @@ Here's a quick guide to help you pick the right vector store for your use case:
 - If you are already utilizing SingleStore, or if you find yourself in need of a distributed, high-performance database, you might want to consider the [SingleStore](./integrations/singlestore) vector store.
 - If you are looking for an online MPP (Massively Parallel Processing) data warehousing service, you might want to consider the [AnalyticDB](./integrations/analyticdb) vector store.
 - If you're in search of a cost-effective vector database that allows run vector search with SQL, look no further than [MyScale](./integrations/myscale).
+- If you're in search of a vector database that can loaded on both browser and server side, look no further than [CloseVector](./integrations/closevector). It aims to running vector database on cross-platform.
```

Comment:
Some typos here - how about:

`- If you're in search of a vector database that you can load from both the browser and server side, check out [CloseVector](./integrations/closevector). It's a vector database that aims to be cross-platform.`

Reply:
Great idea! I edited the docs in the latest commit.